api/FontFaceSetLoadEvent.json

```diff
@@ -72,6 +72,48 @@
           }
         }
       },
+      "worker_support": {
```

Comment:
Note, this interface indicates worker support in the spec: https://drafts.csswg.org/css-font-loading/#fontfacesetloadevent

I mirrored the BCD for chrome as it would all have been added at the same time.

Note, I did not add worker support for the events that were fired.

Reply:
FYI @hamishwillee It looks like mirroring BCD for Chrome was incorrect in this case (see [this PR](https://github.com/mdn/browser-compat-data/pull/24653)).